src/content/data-feeds/starknet/index.mdx

```diff
@@ -1,167 +1,84 @@
 ---
 section: dataFeeds
 date: Last Modified
-title: "Using Data Feeds on StarkNet"
+title: "Using Data Feeds on Starknet"
 isIndex: true
-whatsnext: { "Starknet Price Feed Contract Addresses": "/data-feeds/price-feeds/addresses?network=starknet" }
+whatsnext:
+  {
+    "Starknet Price Feed Contract Addresses": "/data-feeds/price-feeds/addresses?network=starknet",
+    "Deploy and interact with a Consumer Contract using Starknet Foundry": "/data-feeds/starknet/tutorials/snfoundry/consumer-contract",
+  }
 metadata:
-  description: "Using Chainlink Data Feeds on StarkNet"
+  description: "Explore how to use Chainlink Data Feeds on Starknet to power your applications with data."
 ---
 
-StarkNet is a permissionless decentralized ZK-Rollup operating as an L2 network over Ethereum. Unlike other Ethereum L2 networks, StarkNet is not EVM-compatible and uses [Cairo](https://docs.cairo-lang.org/index.html) as its smart contract language. Chainlink Data Feeds are available on the StarkNet testnet as Cairo smart contracts.
+import { CopyText } from "@components"
 
-You can read Chainlink Data Feeds on StarkNet using an [onchain contract](#running-the-onchain-example) that you compile and deploy. Alternatively, you can [read the data feed offchain](#running-the-offchain-example) without a StarkNet account. You can complete these steps using only the [StarkNet CLI commands](https://docs.starknet.io/documentation/tools/CLI/commands/), but the example scripts demonstrate how to compile, deploy, and interact with StarkNet contracts programmatically. [StarkNet.js](https://www.starknetjs.com/), [HardHat](https://hardhat.org/), and the [StarkNet Hardhat Plugin](https://shard-labs.github.io/starknet-hardhat-plugin/) simplify the processes, which normally require you to keep track of your class hashes and ABI files. See [https://docs.starknet.io/](https://docs.starknet.io/documentation/) for more information about writing and compiling Cairo contracts for StarkNet.
+Starknet is a permissionless decentralized ZK-Rollup operating as an L2 network over Ethereum. Unlike other Ethereum L2 networks, Starknet is not EVM-compatible and uses [Cairo](https://docs.cairo-lang.org/index.html) as its smart contract language. Chainlink Data Feeds are available on Starknet Sepolia as Cairo smart contracts.
```

Comment:
```suggestion
Starknet is a permissionless decentralized Zk-rollup operating as an L2 network over Ethereum. Unlike other Ethereum L2 networks, Starknet is not EVM-compatible and uses [Cairo](https://docs.cairo-lang.org/index.html) as its smart contract language. Chainlink Data Feeds are available on Starknet Sepolia as Cairo smart contracts.
```

Aligning with the style from https://chain.link/education-hub/zero-knowledge-rollup